bin/autodetect/CCmdLineParser.cc

```diff
@@ -103,7 +104,9 @@ bool CCmdLineParser::parse(int argc,
                         "Optional file to persist state to - not present means no state persistence")
             ("persistIsPipe", "Specified persist file is a named pipe")
             ("persistInterval", boost::program_options::value<core_t::TTime>(),
-                        "Optional interval at which to periodically persist model state - if not specified then models will only be persisted at program exit")
+                        "Optional time interval at which to periodically persist model state - if not specified then models will only be persisted at program exit. (Mutually exclusive with bucketPersistInterval)")
```

Comment:
> Tying to the bucket length makes more sense to me since the models change on this time scale.

But remember that this is not greenfield functionality in this PR.  We've done background persistence based on wall clock time for about 5 years now.  The rationale for using wall clock time was that if a lookback churns through 1000 half hour buckets in 20 minutes of wall clock time we probably don't need a background persistence in the middle of that work, but if running in real time then we want lots of periodic persists during the 500 hours it takes to get through 1000 half hour buckets.

This really leads on to the question of whether this functionality is only ever going to be used as an internal debugging tool.  At one time the thinking was that it would be internal only.  End users don't call the C++ process directly, so it doesn't matter if the options are esoteric (although it's still best for the descriptions to be factually accurate).  But if the thinking on exposing this externally has now changed then the external user interface needs more careful thought, including how to protect people from accidentally doing a denial-of-service attack on themselves by for example persisting huge model state every 2 buckets during a 10000 bucket lookback.  Also before exposing this externally things like using `jq` from a Python script would need revisiting...